docs_src/microservices/device/profile/V2DeviceProfileRef.md

```diff
@@ -0,0 +1,65 @@
+# V2 Device Profile Reference
+
+This chapter details the structure of a Device Profile and allowable values for
+its fields.
+
+Device Profile
+--------------
+
+Field Name | Type | Required? | Notes
+:--- | :--- | :--- | :---
+name | String | Y | Must be unique in the EdgeX deployment
+manufacturer | String | N |
+model | String | N |
+labels | Array of String | N |
+deviceResources | Array of DeviceResource | Y |
+deviceCommands  | Array of DeviceCommand | N |
+
+DeviceResource
+--------------
+
+Field Name | Type | Required? | Notes
+:--- | :--- | :--- | :---
+name | String | Y | Must be unique in the EdgeX deployment
+description | String | N |
+isHidden | Bool | N | Expose the DeviceResource to Command Service or not, default false
+tag | String | N |
```

Comment:
Add a note  on how this is used.

Reply:
Still missing note

Reply:
this field is actually not used anywhere ... should we remove it here ?

Reply:
Could be used by a custom device service, correct?
@cloudxxx8 , @iain-anderson , what was the intended purposed for this field?

Reply:
it has been there since v1, and we have never used it.
I suggest to remove it, and it won't break anything.
if needed, we should add `tags map[string]string` , and it can be the tags added to event automatically.
string data type is just not enough to use.

Reply:
Works for me.